libs/blocks/section-metadata/section-metadata.js

```diff
@@ -0,0 +1,39 @@
+function handleBackground(div) {
+  const pic = div.querySelector('picture');
+  const section = div.closest('.section');
+  if (pic) {
+    if (section) {
+      section.classList.add('has-background');
+      pic.classList.add('section-background');
+      section.insertAdjacentElement('afterbegin', pic);
+    }
+  } else {
+    const color = div.textContent;
+    if (color && section) {
+      section.style.backgroundColor = color;
+    }
+  }
```

Comment:
nit: you could early return and avoid having to check for section twice:
```suggestion
  const section = div.closest('.section');
  if (!section) return;
  if (pic) {
    section.classList.add('has-background');
    pic.classList.add('section-background');
    section.insertAdjacentElement('afterbegin', pic);
  } else {
    const color = div.textContent;
    if (color) {
      section.style.backgroundColor = color;
    }
  }
```